meta-lmp-bsp/recipes-kernel/kernel-modules/kernel-module-nxp89xx_git.bb

```diff
@@ -12,3 +12,5 @@ S = "${WORKDIR}/git/mxm_wifiex/wlan_src"
 inherit module
 
 EXTRA_OEMAKE = "KERNELDIR=${STAGING_KERNEL_BUILDDIR} -C ${STAGING_KERNEL_BUILDDIR} M=${S}"
+
+INHIBIT_PACKAGE_DEBUG_SPLIT = "1"
```

Comment:
Mind having it based on the modsign feature availability?

e.g.

`INHIBIT_PACKAGE_DEBUG_SPLIT = "${@bb.utils.contains('DISTRO_FEATURES', 'modsign', '1', '', d)}"`

Reply:
Just because without modsign the debug split won't really cause any bad side effect.